### ISSUE TITLE
Missing stacktrace-js file

### ISSUE THREAD

**Issue by PatMurray86** [serenity-js/tutorial-from-scripts-to-serenity#6]
Hi I'm new to using serenity and I'm following your tutorial @ http://serenity-js.org/from-scripts-to-serenity/making-the-tests-speak-for-themselves.html

I created a fresh copy of the repo but when I run the first test it fails,

Actual error message

`Failures:

1) Scenario: Adding an item to a list with other items - features\add_new_items.feature:7
   Step: Given that James has a todo list containing Buy some cookies, Walk the dog - features\add_new_items.feature:9
   Step Definition: node_modules\serenity-js\src\serenity-cucumber\webdriver_synchroniser.ts:47
   Message:
     TypeError: Cannot read property 'fileName' of undefined
         at to_location (C:\Users\Pacina\Desktop\SerenityJS\tutorial-from-scripts-to-serenity\node_modules\@serenity-js\core\src\screenplay\actor.ts:152:22)
         at <anonymous>
         at process._tickCallback (internal/process/next_tick.js:188:7)
     From: Task: <anonymous>
         at World.stepWrapper (C:\Users\Pacina\Desktop\SerenityJS\tutorial-from-scripts-to-serenity\node_modules\serenity-js\src\serenity-cucumber\webdriver_synchroniser.ts:72:18)
         at World.stepWrapper (C:\Users\Pacina\Desktop\SerenityJS\tutorial-from-scripts-to-serenity\node_modules\serenity-js\src\serenity-cucumber\webdriver_synchroniser.ts:104:32)
         at World.arity2 (eval at module.exports (C:\Users\Pacina\Desktop\SerenityJS\tutorial-from-scripts-to-serenity\node_modules\util-arity\arity.js:22:24), <anonymous>:3:45)
         at _combinedTickCallback (internal/process/next_tick.js:131:7)
         at process._tickCallback (internal/process/next_tick.js:180:9)

1 scenario (1 failed)
3 steps (1 failed, 2 skipped)
0m02.279s
[10:26:18] I/local - Shutting down selenium standalone server.
[10:26:18] I/launcher - 0 instance(s) of WebDriver still running
[10:26:18] I/launcher - chrome #01 failed 1 test(s)
[10:26:18] I/launcher - overall: 1 failed spec(s)
[10:26:18] E/launcher - Process exited with error code 1
npm ERR! Test failed.  See above for more details.`

So my failure point is the fileName which is part of the StackFrame I'm missing I believe

Thanks in advance for any help you can give me

P.S
Before this I went through the tutorial @http://serenity-js.org/from-scripts-to-serenity/writing-what-you-would-like-to-read.html#getting-started

And ran the command npm install serenity-js --save and everything worked up to the Asking the right Questions point (following along with the tutorial) and then I was receiving the same error.

**Comment by jan-molak** [comment]
Hi @PatMurray86! Thanks for checking out the tutorial. Serenity/JS is compatible with an LTS (Long-Term Support) version of Node.js, currently 6.11.x. Could you be using a non-LTS version, such as 8?

You can check that by running:

```
node --version
```

**Comment by sabau** [comment]
Hi @jan-molak, I'm facing the same problem. (I'm using Windows 7 64bit)
I had node v6.11.3 and my project worked fine.
I had to upgrade to node v8.6.0, then v8.7.0 and with both versions I incurred in the same error in 

`@serenity-js\core\src\screenplay\actor.ts line 152,22` frame was undefined

My particular case was for the following sequence of actions, I was able to Wait, Click and navigate, but not use Enter:

```javascript
@step('{0} login as  "#username"')
public performAs(actor: PerformsTasks): PromiseLike<void> {
    return actor.attemptsTo(
        Wait.until(LoginPage.targets.loginForm, Is.present()),
        Click.on(LoginPage.targets.usernameField),
        Enter.theValue(this.username).into(LoginPage.targets.usernameField), <--BREAKS HERE
        Click.on(LoginPage.targets.passwordField),
        Enter.theValue(this.password).into(LoginPage.targets.passwordField),
        Click.on(LoginPage.targets.submitButton),
        Wait.upTo(new Duration(this.timeout)).until(BasePage.targets.spinner, Is.invisible())
    );
}
```

I solved by downgrading to node v6.11.4 and everything is working fine again.

Hope it can be useful, I'll wait for updates before upgrading again/if I'll find the time to investigate I'll try to look deeper.

Sorry, this does not regard the tutorial only, is more generic issue.

**Comment by AnnikaNissen** [comment]
I had exactly the same problem as PatMurray86 when getting startedt with the tutorial using Node 8.9.1 (which now is the current LTS). After downgrading to Node 6.11.5 it worked fine. Are there plans regarding support for Node 8.X? 

In the mean time, a hint in the documentation would be useful. I find it really annoying trying something with a tutorial and then figuring out that it doesn't work.

**Comment by jan-molak** [comment]
Hi @AnnikaNissen - I'm sorry you felt that way. Yes, I'm planning to look into why one of the Serenity/JS dependencies (`stacktrace-js` ) seems to be incompatible with Node 8.x.

If you believe that the problem is not isolated to the tutorial but affects Serenity/JS itself - please [file an issue](https://github.com/jan-molak/serenity-js) describing your configuration and steps to reproduce it.

If you'd like to take a stab at a pull request or improving the documentation I'd be happy to support you - this is a free open-source project and your help is greatly appreciated!

Jan

**Comment by sabau** [comment]
Hi, I can confirm that is not affecting only the tutorial. I had this issue on our pre-production tests

**Comment by jan-molak** [comment]
This problem has been resolved in jan-molak/serenity-js#122.
Every CI build of Serenity/JS is now also executed against [both node 6 and node 8](https://github.com/jan-molak/serenity-js/blob/1e9fa93d35e26f8587c6d38171d1cdd117c9c072/.travis.yml#L5) to ensure compatibility.

Thanks for using Serenity/JS and please let me know if you come across any other issues!
Jan
